src/components/dashboard/sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,37 @@
+import 
+{BsGrid1X2Fill, BsPeopleFill, BsFillGearFill}
+ from 'react-icons/bs'
+ import "../../../pages/DashboardPage.css"
+ import "./Sidebar.css"
+ import logo from "../../../assets/dashboard/logo.svg"
+
+function Sidebar({openSidebarToggle, OpenSidebar}) {
```

Comment:
To declare these props as the above prop type we can do:
`function Sidebar({openSidebarToggle, OpenSidebar}: Props) {`